app/assets/javascripts/discourse/tests/acceptance/sidebar-user-test.js

```diff
@@ -9,24 +9,6 @@ import {
 } from "discourse/tests/helpers/qunit-helpers";
 import Sinon from "sinon";
 
-acceptance(
-  "Sidebar - Logged on user - Legacy navigation menu enabled",
```

Comment:
No need for this test anymore since the legacy navigation menu no longer exists.